ui/components/show/showCards/ShowCards.tsx

```diff
@@ -12,72 +11,74 @@ import CardImgContainer from './ImgContainer/CardImgContainer';
 import TagsContainer from '../../layout/cardElements/TagsContainer/TagsContainer';
 import ContentSection from '../../layout/ContentSection/contentSection';
 
-import styles from "./ShowCards.module.scss";
+import { Swiper, SwiperSlide } from 'swiper/react';
```

Comment:
```suggestion
import { Swiper, SwiperSlide } from 'swiper/react';
import { Pagination, A11y } from 'swiper/modules';
```
Let's add the A11y module so we get some increased accessibility. 

Let's also try the Pagination module, as it might help the user understand this content is scroll-able.